services/app-api/utils/formTemplates/formTemplates.ts

```diff
@@ -325,3 +330,25 @@ const makePCCMTemplateModifications = (reportTemplate: ReportJson) => {
   }
   programTypeQuestion.props!.disabled = true;
 };
+
+export const makeTopicXIIModifications = (reportTemplate: ReportJson) => {
+  /*
+   *   Find Question C1.XII.4, nest 11.a within 10.b
+   *   also remove the response "No deficiencies identified" under question 11
+   */
+  const topicXIIchoices =
+    reportTemplate.routes[2].children![6].form!.fields[0].props?.choices[1];
+  const question11 = topicXIIchoices.children.find(
```

Comment:
`topicXIIchoices` could be `undefined` because of `props?`. If it's not possible for it be `undefined`, should change to `props!` to make that clear.